R/taxonomic-coverage-element.R

```diff
@@ -30,23 +30,23 @@
 #' 
 #' Select "chinook" to append the taxonomy of Oncorhynchus tshawytscha 
 #' (Also known as Chinook Salmon or King Salmon) from the ITIS database. 
```

Comment:
I think this could be condensed into a bulleted list. Something like...

The following frequently cited species are available for convenience:
* `EDIutils::CVPIA_common_species$steelhead` - Oncorhynchus mykiss (ITIS 161989)

Reply:
Agreed! I changed to bulleted list.